### ISSUE TITLE
Add configuration not working - react native tools

### ISSUE THREAD

**Issue by springcoil** [Microsoft/vscode-react-native#985]
Issue Type: <b>Bug</b>

Hi, I wanted to use React-native debugger like this tutorial - https://github.com/Microsoft/vscode-react-native/blob/master/README.md#debugging-react-native-applications. However I'm running into problems getting it to work. I click on 'Add Configuration' in launch.json and nothing happens. I see no change. I expected that this would offer me some options. https://github.com/Microsoft/vscode/issues/63242 is a related issue I believe.

Extension version: 0.9.2
VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:22:55.268Z)
OS version: Darwin x64 18.5.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 2|
|Memory (System)|16.00GB (1.72GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details>

**Comment by ruslan-bikkinin** [comment]
Hi @springcoil and thanks for reaching us. Is `Add configurations` button grayed out in your case?

**Comment by springcoil** [comment]
Hi @ruslan-bikkinin no it's not greyed out. I've got a screenshot. 
<img width="1008" alt="Screenshot 2019-04-23 13 41 55" src="https://user-images.githubusercontent.com/983944/56581770-9c825900-65cd-11e9-8d29-c456a7eb0090.png">

**Comment by ruslan-bikkinin** [comment]
Usually it should be gray when `launch.json` is malformed. In your case just add these lines to `launch.json` to make it work again:
```js
{
    "version": "0.2.0",
    "configurations": [
    ]
}
```

**Comment by springcoil** [comment]
Thanks so much. I'm happy with that resolution. Can I open a PR to just add this to the FAQs?

**Comment by ruslan-bikkinin** [comment]
@springcoil I believe this is a regression in VS Code issue because `Add configuration` button supposed to be disabled in the past in cases like yours. I'd suggest you to let VS Code team know about it, as well, unless there is similar issue opened already.

**Comment by Chuk116** [comment]
I am having a similar issue that I can't quite figure out. I have the react-native tools extension downloaded on VSCode but when I click on the Add Configuration button I don't see any options for React Native. Additionally, when I click on the "Show all automatic debug configurations" in the Run And Debug side panel, I see the "React Native" option like this: 
<img width="1782" alt="Screenshot 2024-05-22 at 5 01 52 PM" src="https://github.com/microsoft/vscode-react-native/assets/28410610/bec59f3b-005c-4c99-a2aa-7100f7088c4f">
However, when I click the React Native option the dropdown is empty like this: 
<img width="1791" alt="Screenshot 2024-05-22 at 5 02 22 PM" src="https://github.com/microsoft/vscode-react-native/assets/28410610/6714f979-140a-47f5-acad-d1df3b1f95b4"> I feel like I have tried every solution that I have found on the web for related issues but nothing has worked so far. Any ideas for other possible solutions?
